server/api/user/order.js

```diff
@@ -27,7 +27,7 @@ router.get('/all', requireUserToken, async (req, res, next) => {
 router.get('/current', requireUserToken, async (req, res, next) => {
   try {
     const { user } = req;
-    const order = await user.currentOrder();
+    const order = await user.currentOrder(); // grab subtotal too?
```

Comment:
Yeah, good idea. There's an instance method, so you could add this below before sending to the client
```javascript
order.subtotal = await order.getSubtotal();
```